data-esnext.js

```diff
@@ -1686,6 +1686,26 @@ exports.tests = [
     }
   ]
 },
+{
+  name: 'template literal revision',
+  category: 'candidate (stage 3)',
+  significance: 'small',
+  link: 'https://github.com/tc39/proposal-template-literal-revision',
+  exec: function() {/*
+    function fn(strings, a, b) {
+      return typeof strings[0] === "undefined" &&
+        typeof strings[1] === "undefined" &&
+        typeof strings[2] === "undefined" &&
```

Comment:
`\u{55}` is a legal escape sequence, both before and after this proposal so you shouldn't be asserting it is undefined